android-core/src/main/java/com/mparticle/networking/MParticleBaseClientImpl.java

```diff
@@ -110,6 +111,11 @@ protected MPUrl getUrl(Endpoint endpoint, @Nullable String identityPath, boolean
             url = domainMappingUrl;
         }
         Uri uri;
+        Logger.debug("before Mansi -->"+url);
```

Comment:
@Mansi-mParticle These two logs should be removed or the message should at least be change to something more generic and informative